app/MMB/src/components/ScrollIntoView.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+
+import * as React from 'react';
+import { View, ScrollView } from 'react-native';
+
+type Props = {|
+  +shouldScrollIntoView: boolean,
+  +scrollToY: (y: number) => void,
+  +children: React.Node,
+|};
+
+type OnLayoutEvent = {|
+  nativeEvent: {
+    layout: { x: number, y: number, width: number, height: number },
+  },
+|};
+
+export class ScrollIntoView extends React.Component<Props> {
+  static defaultProps = {
+    scrollToY: () => {},
```

Comment:
this prop is mandatory, why do we have defaultProp?

Reply:
I get the following Flow error
```
Cannot create ScrollIntoView element because property scrollToY is missing in props [1] but exists in
Props [2].
```
if it is not present. This is because this prop is injected.

Reply:
Then I would set the prop to not be mandatory, since you actually create the component without it.